### ISSUE TITLE
Inconsistent whitespace around `/` in binaries

### ISSUE THREAD

**Issue by ehamberg** [old-reliable/steamroller#16]
When formatting binaries, there is no whitespace around `/` (e.g. in `<<Foo/binary>>`), but if the value before `/` is surrounded by parentheses, the `/` is surrounded by whitespace. Here is an example of where this is happening:

```diff
-base62_encode(I, Acc) when I < 62 ->
-    <<(base62_encode_char(I))/integer, Acc/binary>>;
+base62_encode(I, Acc) when I < 62 -> <<(base62_encode_char(I)) / integer, Acc/binary>>;
```

(Thanks for creating steamroller! Super happy to see stand-alone, opinionated formatters for Erlang!)

**Comment by dtip** [comment]
Thanks for the report! This should be pretty easy to fix.

**Comment by dtip** [comment]
Fixed on master - will be in release 0.13.2
